wheel_left_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "wheel_left_joint"
  pid: {p: 100.0, i: 0.01, d: 10.0}

wheel_right_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "wheel_right_joint"
  pid: {p: 100.0, i: 0.01, d: 10.0}
